apps/fluent-tester/src/E2E/ContextualMenu/pages/ContextualMenuPageObject.win.ts

```diff
@@ -2,10 +2,55 @@ import {
   CONTEXTUALMENU_TESTPAGE,
   CONTEXTUALMENU_TEST_COMPONENT,
   HOMEPAGE_CONTEXTUALMENU_BUTTON,
+  CONTEXTUALMENUITEM_TEST_COMPONENT,
 } from '../../../FluentTester/TestComponents/ContextualMenu/consts';
 import { BasePage, By } from '../../common/BasePage.win';
 
+/* This enum gives the spec file an EASY way to interact with SPECIFIC UI elements on the page.
+ * The spec file should import this enum and use it when wanting to interact with different elements on the page. */
+export const enum ContextualMenuSelector {
+  ContextualMenu = 0, //this._primaryComponent
```

Comment:
nit: the = 0 seems unnecessary, is there a particular reason for it?

Reply:
It's just a visual representation of the numeric value of each enum entry. ContextualMenu would be initialized with the value 0 regardless, but for people who aren't too familiar with JS I thought it might be helpful to have.

You can see [more here  ](https://www.typescriptlang.org/docs/handbook/enums.html). I was always taught this method so I've carried it with me :)